src/negocio/ControladorEmpregado.java

```diff
@@ -19,10 +22,48 @@ public void cadastrarEmpregado() {
 
     }
 
-    public boolean autenticarEmpregado(Empregado empregado) {
-        boolean valido = false;
+    public boolean autenticarEmpregado(String _email, String _senha) {
+        boolean emailValidado = false;
+        boolean validado = false;
+        Empregado eSenha = null;
+        String stringSenha = " ";
 
-        return valido;
+        for(Empregado e : this.repoEmpregado.listar()){
+            if(e.getEmail().equals(_email)){
+                eSenha = e;
+                emailValidado = true;
+            }
+        }
+
+        if(emailValidado == true){
+            //Digest e senha
+            StringBuilder senhaHex = new StringBuilder();
+
+            try{
+                MessageDigest algoritmoEncrypt = MessageDigest.getInstance("SHA-256");
+                byte[] senhaDigest = algoritmoEncrypt.digest(_senha.getBytes(StandardCharsets.UTF_8));
+                for(byte b :  senhaDigest){
+                    senhaHex.append(String.format("%02X",0xFF &b));
+                }
+            }catch (NoSuchAlgorithmException e){
+                e.printStackTrace();
+                return false;
+            }
+
+            eSenha.setSenha(senhaHex.toString());
+            stringSenha = eSenha.getSenha();
```

Comment:
Aqui você pode simplificar para
```suggestion
            stringSenha = senhaHex.toString();
```